snuba/datasets/events.py

```diff
@@ -336,5 +336,5 @@ def _tags_expr(self, column_name, body):
             # to re-use as we won't need it.
             return 'arrayJoin({})'.format(key_list if k_or_v == 'key' else val_list)
 
-    def get_query_schema(self):
-        return EVENTS_QUERY_SCHEMA
+    def get_query_extensions_schema(self):
+        return EVENTS_QUERY_EXTENSIONS_SCHEMA
```

Comment:
could you please provide an empty implementation for outcomes and groupedmessage so that they do not blow up as soon as we start querying them? Those dataset are already enabled even though none is querying them.

Reply:
This is already broken because the query execution right now has a hard dependency these fields (time series extension due to [the query splitter](https://github.com/getsentry/snuba/blob/master/snuba/split.py) and [rate limiting](https://github.com/getsentry/snuba/blob/master/snuba/util.py#L356).) I think we should fix that first so we're not splitting queries on columns that don't actually exist. If anybody was using them, I'd probably be a little more careful here, but I think they *should* be broken as-is because the underlying execution path makes assumptions about the data that aren't valid?

Reply:
"turbo" and "consistent" attributes should work as they are for both outcomes and groupedmessage. Actually I think they should work for all MergeTable datasets.
None is using them yet, but it will be a matter of days before we start at least for outcomes.

Reply:
I'm fine with moving both of those to general use extensions. I'd prefer in the future if those were opt-in on the dataset than inherited, though. I don't think we want to commit to every dataset having the same write path properties that allow us to use `consistent`.

This doesn't really change the fact though that these queries aren't going to work correctly until the splitter is modified, though.

Reply:
Ok, so here there are two separate problems (which do not seem depending on each other).
1) time series issues during split. This is already broken now and it definitively doe not belong to this PR to fix it.
2) consistent and turbo. These should still be extensions but should be used by our three datasets today. Those work properly today. Up to you if you want to add this support to groupedmessage and outcomes. Otherwise I can do it (ok, Manu probably will do that) when he starts querying outcomes. Both solutions are fine.